test/lint-commits.sh

```diff
@@ -2,7 +2,7 @@
 
 set -e
 
-if [[ -z "$TRAVIS" && -z "$JENKINS" ]]; then
+if [[ ! -z "$TRAVIS" ]]; then
```

Comment:
The idea of this if statement was not to run if it detected that it was running on either Travis or Jenkins, but the new negated version will run this only if it's running on Travis. I don't think that will work. What was your idea behind this change?

Reply:
Ah I misunderstood you then, I thought you wanted to have it run on Travis, but not on Jenkins for the moment.

The statement you have will run it if `$TRAVIS` and `JENKINS` are empty, which was always the case, as the env variables were not passed to docker.